integration/tests/cook/test_basic.py

```diff
@@ -213,19 +215,32 @@ def test_max_runtime_exceeded(self):
         job_timeout_interval_ms = job_timeout_interval_seconds * 1000
         max_runtime_ms = 5000
         assert max_runtime_ms < job_timeout_interval_ms
-        job_uuid, resp = util.submit_job(self.cook_url, command='sleep %s' % job_timeout_interval_seconds,
-                                         executor=job_executor_type, max_runtime=max_runtime_ms)
-        self.assertEqual(201, resp.status_code, msg=resp.content)
-        util.wait_for_job(self.cook_url, job_uuid, 'completed', job_timeout_interval_ms)
-        job = util.load_job(self.cook_url, job_uuid)
-        self.assertEqual(1, len(job['instances']))
-        self.assertEqual('failed', job['instances'][0]['status'])
-        self.assertEqual(2003, job['instances'][0]['reason_code'])
-        self.assertEqual('Task max runtime exceeded', job['instances'][0]['reason_string'])
-        if job_executor_type == 'cook':
-            job = util.wait_for_exit_code(self.cook_url, job_uuid)
-            self.assertEqual(-15, job['instances'][0]['exit_code'])
-            self.assertTrue(bool(job['instances'][0]['sandbox_directory']))
+        try:
+            job_uuid, resp = util.submit_job(self.cook_url,
+                                             command=f'sleep {job_timeout_interval_seconds}',
+                                             executor=job_executor_type, max_runtime=max_runtime_ms)
+            self.assertEqual(201, resp.status_code, msg=resp.content)
+            util.wait_for_job(self.cook_url, job_uuid, 'running')
+            util.wait_for_job(self.cook_url, job_uuid, 'completed', job_timeout_interval_ms)
+            job = util.wait_for_end_time(self.cook_url, job_uuid)
+            self.assertEqual(1, len(job['instances']))
```

Comment:
Can we include the job details in the assertion message. `f"Job details: {json.dumps(job, sort_keys=True)}"` ?